Tests/feaLib/data/spec4h1.fea

```diff
@@ -43,16 +43,15 @@ feature liga {
     sub c k by c_k;
 
     # The rules above will be placed in a lookup that is registered
-    # only under the script latn, language DEU.
+    # only under the script latn, language DEU, along with the lookup
+    # registered for the dflt language for this script.
 
     language TRK;
 
-    # This will inherit both the top level default rules - the rules
-    # defined before the first 'script' statement, and the
-    # script-level default rules for 'latn': all the lookups of this
-    # feature defined after the 'script latn' statement, and before
-    # the language DEU statement.  If TRK were not named here, it
-    # would not inherit the default rules for the script latn.
+    # This will inherit only the lookup defined for the dflt language
+    # for this script.
+    # If this statement were not here, the language would only inherit
+    # the lookup at the start of this feature before script latn.
```

Comment:
The wording of these comments is copied verbatim from the Feature File spec (example 1 from paragraph 4h).
We can't simply unilaterally change it.